src/sentry/integrations/slack/utils.py

```diff
@@ -381,7 +390,10 @@ def get_channel_id_with_timeout(integration, name, timeout):
                 # Slack limits the response of `<list_type>.list` to 1000 channels
                 params=dict(payload, cursor=cursor, limit=1000),
             )
+            items.raise_for_status()
```

Comment:
What are the implications of raising here? My guess is that if we get a non-200, this will raise, and then we won't instrument the response (cause it won't reach the next line).

Do we need to `items.raise_for_status()`?

Reply:
@manuzope this is a really good point I overlooked - `raise_for_status` is used in a couple other places so I moved some stuff around to be able to do both, and removed it from this particular method.